wp-content/plugins/core/src/Assets/Admin_Assets_Enqueuer.php

```diff
@@ -25,4 +30,34 @@ public function register(): void {
 		);
 	}
 
+	public function enqueue_login_styles(): void {
+		$args = $this->get_asset_file_args( $this->assets_path . self::LOGIN_ASSETS_FILE );
+
+		wp_enqueue_style(
+			self::LOGIN,
+			$this->assets_path_uri . self::LOGIN_FILE_NAME . '.css',
+			[],
+			$args['version'] ?? false,
+			'all',
+		);
+	}
+
+	public function update_login_header_url(): string {
+		return get_bloginfo( 'url' );
+	}
+
+	public function update_login_header(): void {
+		$login_logo_id = get_field( Login_Settings::LOGIN_LOGO, 'option' );
+		$login_logo    = $login_logo_id ? wp_get_attachment_image_src( $login_logo_id )[0] : false;
+
+		if ( $login_logo === false ) {
+			return;
+		}
+
+		echo sprintf(
+			'<style>body.login #login h1 a {background: url(%s) no-repeat center top transparent;background-size: 200px 44px;height: 44px;width: 200px; margin: 0 auto 45px auto;} body.login #login {padding-top: 86px;}</style>',
```

Comment:
Curious how you landed on these specific pixel dimensions for the bg image? I wonder if we should leave the bg image size to "contain" instead of setting it specifically? Otherwise, I'm worried that a logo with a different A/R will get skewed.

Reply:
The pixel dimensions are based off of the instructions text from the ACF field, just smaller, but good call on the background size, it should have been switched to contain (I copied from some old code I had laying around where the image didn't change so it was specifically set to a size and then didn't think to change it).